src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -51,7 +51,12 @@ const Body = BaseFormWithPolling.extend(
       return this.options.currentViewState.relatesTo.value;
     },
 
-    doLoopback(authenticatorDomainUrl = '', ports = [], challengeRequest = '') {
+    doLoopback(deviceChallenge) {
+      let authenticatorDomainUrl = deviceChallenge.domain !== undefined ? deviceChallenge.domain : '';
+      let ports = deviceChallenge.ports !== undefined ? deviceChallenge.ports : [];
+      let challengeRequest = deviceChallenge.challengeRequest !== undefined ? deviceChallenge.challengeRequest : '';
+      let probeTimeoutMillis = deviceChallenge.probeTimeoutMillis !== undefined ?
+        deviceChallenge.probeTimeoutMillis : 100;
```

Comment:
This logic is extremely similar to the `doLoopback` function in `ChallengeOktaVerifyFastPassView`. Can we reuse this logic via a named function export?